xarray/core/dataarray.py

```diff
@@ -1375,6 +1375,7 @@ def chunk(
                 "Supplying chunks as dimension-order tuples is deprecated. "
                 "It will raise an error in the future. Instead use a dict with dimensions as keys.",
                 category=DeprecationWarning,
+                stacklevel=2,
```

Comment:
let's use `utils.emit_user_level_warning`

Reply:
Great. Do you agree with the change though? :)

Reply:
yes!